stack.yaml

```diff
@@ -44,7 +44,9 @@ extra-deps:
 - cryptostore-0.2.1.0
 - happy-1.19.12
 - hspec-wai-json-0.10.1
-- ipfs-1.0.3
+# - ipfs-1.0.3
+- git: https://github.com/fission-suite/ipfs-haskell.git
+  commit: 0b52e01fc53f05912350a413be6e246bc8d0e919
```

Comment:
Should we publish this? 👀

Reply:
yup! but probably want a review first: https://github.com/fission-suite/ipfs-haskell/pull/15

Reply:
Approved